src/test/resources/features/campaign.feature

```diff
@@ -2,19 +2,27 @@ Feature: Requests for Campaign endpoint
 
   @GetCampaigns
   Scenario: Get all Campaigns
-    Given
+  Given
     When I add this "/Campaign" endpoint and send the request
```

Comment:
could be "I set a GET  request's endpoint to "/Campaign" and execute the request"